koku/api/report/test/util/constants.py

```diff
@@ -78,6 +89,9 @@ def __setitem__(self, item, value):
         "resource_ids": AWS_RESOURCE_IDS,
         "units": AWS_UNITS,
         "cost_category": AWS_COST_CATEGORIES,
+        "operating_systems": AWS_OPERATING_SYSTEMS,
```

Comment:
I think it should be operating_system, in singular.

Reply:
using plural here because we are referring multiple choices, there are other examples a few lines above in the `AWS_CONSTANTS` like:

```
"product_codes": AWS_PRODUCT_CODES,
"product_families": AWS_PRODUCT_FAMILIES,
```